### ISSUE TITLE
Unrecognized key "skip" in chromatic.config.json

### ISSUE THREAD

**Issue by hulk510** [chromaui/chromatic-cli#889]
Hi, there.
The issue is that the "skip" option is not being recognized correctly in the chromatic.config.json file. I have attempted to find a list of parameters that can be specified in this file, but was unable to find any. My understanding is that the settings from [the CLI options](https://www.chromatic.com/docs/cli/#configuration-options) should be directly applicable in the chromatic.config.json file, but please correct me if I'm mistaken.

The specific error log is as follows:

```bash
If you need help, please chat with us at https://www.chromatic.com/docs/cli for the fastest response.
You can also email the team at support@chromatic.com if chat is not an option.

Please provide us with the above CLI output and the following info:
{
  "timestamp": "2024-01-15T09:21:50.582Z",
  "sessionId": "03c3c59c-86f8-4616-920c-87ff61de9be2",
  "nodePlatform": "darwin",
  "nodeVersion": "18.17.1",
  "packageName": "chromatic",
  "packageVersion": "10.2.0",
  "flags": {
    "outputDir": [],
    "storybookBuildDir": [],
    "externals": [],
    "onlyStoryFiles": [],
    "onlyStoryNames": [],
    "untraced": [],
    "fileHashing": true,
    "interactive": true,
    "appCode": []
  },
  "errorType": "Error",
  "errorMessage": "✖ Configuration file chromatic.config.json was invalid, please check the allowed keys."
}

Error: ✖ Configuration file chromatic.config.json was invalid, please check the allowed keys.

- Unrecognized key(s) in object: 'skip'

```

It seems that the 'skip' option might not have been added in [this section](https://github.com/chromaui/chromatic-cli/blob/a13eb7a906cd310bd1bfbc39221e40e85ecb944e/node-src/lib/getConfiguration.ts#L9-L35C1)?
I have confirmed that the skip option is available in the CLI and other interfaces.

Thank you for your attention to this matter.

**Comment by ghengeveld** [comment]
Thanks for raising this. It's an oversight in the configuration schema. I'll add it.

**Comment by ghengeveld** [comment]
Addressed here: https://github.com/chromaui/chromatic-cli/pull/890

**Comment by ghengeveld** [comment]
:rocket: Issue was released in [`v10.3.0`](https://github.com/chromaui/chromatic-cli/releases/tag/v10.3.0) :rocket:
